File: MH/DAY7/1476.js
//메모리 초과가 뜨는데 풀이 과정은 맞은 문제 - 완전탐색 
const path = process.platform === "linux" ? "/dev/stdin" : "../input.txt";
let input = require("fs").readFileSync(path).toString().trim().split(" ");

let x = 1;
while (true) {
  x++;
  if (
    (x - input[0]) % 15 === 0 &&
    (x - input[1]) % 28 === 0 &&
    (x - input[2]) % 19 === 0
  ) {
    console.log(x);
    process.exit();
  }
}

File: KSY/DAY5/1476.py
E, S, M = map(int, input().split())
result, e, s, m = 1, 1, 1, 1

while True:
    if e == E and m == M and s == S:
        break
    e += 1
    m += 1
    s += 1
    result += 1
    if e == 16:
        e = 1
    if m == 20:
        m = 1
    if s == 29:
        s = 1

print(result)
